site/docs/components/toast-notification.mdx

```diff
@@ -56,30 +56,29 @@ When you are notifying a user of the outcome of an action they have taken, provi
 *   **Cautionary**
     *   TODO: Add images and examples here
 
-### When not to use?
+## When not to use
 
-*   When you are speaking directly to a piece of content, use [Inline Notifications](https://drive.google.com/a/cultureamp.com/open?id=1harIX82v35t4Qu9RwFOAjk5f6zFQ509hYQvQhV3857A) which can be positioned closer to it.
-*   When a message applies at a system level or does not apply to a specific piece of content and it needs to alert all users, use [Global Notifications](https://drive.google.com/a/cultureamp.com/open?id=1-31stGiBx58FygpLFsA4IBiFfB48YxeVY8S9Q1WCxe0).
-*   When an action is user triggered and you need to inform the user, use a [Confirmation Modal](https://drive.google.com/a/cultureamp.com/open?id=1xjuMUCQLogH6l61hq1KbSCcp9BPifr7xG1av6QOsZKI) with relevant [Mood](https://drive.google.com/a/cultureamp.com/open?id=1o3NevxWj3xU2wvrJQJyZ5ZggNDobuRquqVS5jldDu9g), e.g. Informative Confirmation Modal
+*   When you are speaking directly to a piece of content, use [Inline Notifications](/components/inline-notification) which can be positioned closer to it.
+*   When a message applies at a system level or does not apply to a specific piece of content and it needs to alert all users, use [Global Notifications](/components/global-notification).
+*   When an action is user triggered and you need to inform the user, use a [Confirmation Modal](/components/modal) with relevant [Mood](/guidelines/mood), e.g. Informative Confirmation Modal
 
-### See also…
+## See also
 
-*   **Inline notification**: [Inline notifications](https://docs.google.com/document/d/1harIX82v35t4Qu9RwFOAjk5f6zFQ509hYQvQhV3857A/edit#heading=h.2ocvpvil9swi) appear within content areas as close as possible to the thing that it’s about. It is used for contextual information. \
+*   **Inline notification**: [Inline notifications](/components/inline-notification) appear within content areas as close as possible to the thing that it’s about. It is used for contextual information. \
 
-*   **Global Notification: **A [Global Notification](https://drive.google.com/a/cultureamp.com/open?id=1-31stGiBx58FygpLFsA4IBiFfB48YxeVY8S9Q1WCxe0) is a message object used for system-level notifications that inform the user about system status such as logging in and background processes like data imports. \
+*   **Global Notification: **A [Global Notification](/components/global-notification) is a message object used for system-level notifications that inform the user about system status such as logging in and background processes like data imports. \
```

Comment:
These backspaces at the end of the lines here I'm guessing are an artefact from a find + replace.